src/main/java/Person.java

```diff
@@ -0,0 +1,19 @@
+public class Person {
+
+    private String name;
+
+    public Person(String n)
+    {
+        name = n;
+    }
+
+    public String IsBob() {
```

Comment:
Methods start with a lower case letter.

Methods prefixed with the word `is`, `does`, `can`, `will` presume a `boolean` return-type.
(For example, one might expect `isBob()` to return `true` when the object's name is `"Bob"`)